pandas/core/arrays/period.py

```diff
@@ -113,12 +115,23 @@ def freq(self, value):
 
     _attributes = ["freq"]
 
+    def __new__(cls, values, freq=None, **kwargs):
```

Comment:
maybe add a todo comment to convert this to init once the Mixin is converted to proper array